src/input_common/sdl/sdl_impl.cpp

```diff
@@ -48,6 +48,27 @@ static int SDLEventWatcher(void* userdata, SDL_Event* event) {
     return 0;
 }
 
+static const std::array<SDL_GameControllerButton, Settings::NativeButton::NumButtons>
```

Comment:
```suggestion
constexpr std::array<SDL_GameControllerButton, Settings::NativeButton::NumButtons>
```

const/constexpr file-scope variables already have internal linkage by default.